### ISSUE TITLE
Can you share the Loss Curve?

### ISSUE THREAD

**Issue by necrophagists** [TinyLLaVA/TinyLLaVA_Factory#47]
Hello,can you share the pretrain Loss Curve and fine-tune Loss Curve? I have some questions about my reproduction results.Thank you!

**Comment by cooleel** [comment]
@baichuanzhou Could u please share the loss curve in pretraining and finetuning? Thanks.
